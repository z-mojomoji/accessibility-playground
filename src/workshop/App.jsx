import React from 'react';
import { BrowserRouter, Route } from 'react-router-dom';
import { Form, Checkbox, Image, Icon, Link, Button, Label, Radio, Input } from 'workshop';
// import '@agoda/kite/desktop';

const App = () => (
    <BrowserRouter>
        <div>
            <main>
                <Route exact path="/Image" component={Image} />
                <Route exact path="/Icon" component={Icon} />
                <Route exact path="/Link" component={Link} />
                <Route exact path="/Button" component={Button} />
                <Route exact path="/Label" component={Label} />
                <Route exact path="/Input" component={Input} />
                <Route exact path="/Checkbox" component={Checkbox} />
                <Route exact path="/Radio" component={Radio} />
                <Route exact path="/Form" component={Form} />
            </main>
        </div>
    </BrowserRouter>
);

export default App;
