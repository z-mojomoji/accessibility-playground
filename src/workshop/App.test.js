import React from 'react';
import { render } from 'react-dom';
import renderer from 'react-test-renderer';

import App from './App';

describe('<Router />', () => {
  it('Should render without crashing', () => {
    const div = document.createElement('div');
    render(<App />, div);
  });

  it('renders a snapshot', () => {
    const tree = renderer.create(<App/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

