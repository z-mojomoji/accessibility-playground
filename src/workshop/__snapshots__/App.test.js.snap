// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Router /> renders a snapshot 1`] = `
<div>
  <main />
</div>
`;
